foreshadow/foreshadow.py

```diff
@@ -373,7 +377,7 @@ def dict_serialize(self, deep=False):
 
     @staticmethod
     def _customize_serialized_estimator(estimator):
-        if isinstance(estimator, MetaEstimator):
+        if isinstance(estimator, EstimatorWrapper):
```

Comment:
I think `estimator` can never be an `EstimatorWrapper` anymore, since line 373 calls this function with `self.estimator`.